doc/sphinx-guides/source/installation/config.rst

```diff
@@ -1783,6 +1838,21 @@ To facilitate large file upload and download, the Dataverse Software installer b
 
 and restart Payara to apply your change.
 
+mp.config.profile
++++++++++++++++++
+
+MicroProfile Config 2.0 defines the `concept of "profiles" <https://download.eclipse.org/microprofile/microprofile-config-2.0/microprofile-config-spec-2.0.html#configprofile>`_.
+They can be used to change configuration values by context. This is used in Dataverse to change some configuration
+defaults when used inside container context rather classic installations.
+
+As per the spec, you will need to set the configuration value ``mp.config.profile`` to ``ct`` as early as possible.
```

Comment:
What does "ct" stand for?

Reply:
"ct" means "container". I've been using this in quite a few places now, e.g. the Maven profile to build the images etc.

Reply:
Ok, thanks.